onnxruntime/core/providers/cpu/ml/ml_common.h

```diff
@@ -224,18 +224,22 @@ static inline void multiclass_probability(int64_t classcount, const std::vector<
   }
 }
 
-static inline float ml_logit(float val) {
+static const float ml_sqrt2 = 1.41421356f;
+
+static inline float ml_logistic(float val) {
   float v = 1 / (1 + std::exp(-std::abs(val)));
   return (val < 0) ? (1 - v) : v;
 }
 
+static inline float ml_probit(float val) {
```

Comment:
Function name does not comply with the style guide. This applies to many other functions as well.

Reply:
Yes, most of the algorithms in this directory would benefit from a scrub 👍. This PR was mostly to add the missing function. We should create another PR to refactor most of the algorithms, and also add comprehensive unit tests (test coverage is low in general in this folder).

Reply:
BTW, the need of a probit function is also not very clear -- it assumes that some tree model generates a probability value directly, backed by a score whose distribution follows the standard normal Gaussian, and we'd like to retrieve the score again. If we need such a function, it seems like a separate operator might be better.

Reply:
Sure, functions that existed before this PR can be rectified later. But I guess ml_probit is newly introduced. At least for the new functions, let's follow the style guide.

Reply:
Updated several functions to comply with style guide.